File: tests/integration/features/interface-public/interface-public.java
// Interface members without an explicit access modifier default to
// <public/>. This makes implicit visibility explicit — a simple
// //method[public] query hits every visible interface method.

interface Shape {
    double area();              // implicitly public
    double perimeter();         // implicitly public
    default String name() {     // implicitly public
        return "shape";
    }
    public void stroke();       // explicit — should still be <public/>
}

File: tests/integration/features/interface-public/interface-public.cs
// Interface members without an explicit access modifier default to
// <public/>. Matches Java behavior — C# semantics agree.

interface IShape
{
    double Area();                    // implicitly public
    double Perimeter();               // implicitly public
    string Name => "shape";           // implicitly public (expression-bodied property)
    public void Stroke();             // explicit — still <public/>
}
